playwright_tests/core/basepage.py

```diff
@@ -14,14 +14,14 @@ def _get_element_locator(self, xpath: str, with_wait=True) -> Locator:
         This helper function returns the element locator from a given xpath.
         """
         if with_wait:
-            self.page.wait_for_selector(xpath)
+            self.__wait_for_dom_load_to_finnish()
```

Comment:
Spelling typo in method name here and in a few other places.
```suggestion
            self.__wait_for_dom_load_to_finish()
```

Reply:
Ups!! Thanks Ryan!

Fixed.